src/server/views/dashboard/users-edit.njk

```diff
@@ -8,10 +8,6 @@
   <h1 class="govuk-heading-l">Editing User</h1>
   <h4 class="govuk-heading-s">{{ user.email }}</h4>
 
-  {% if userSaved %}
```

Comment:
@RichardBray I've removed userSaved which was being passed as a render parameter. I've refactored to req.flash in users-list.njk. 

I've also redirected the user back. You can only make someone a superAdmin. After that you won't be able to revoke or change anything, so no point in keeping them on that page.